### PR TITLE
fix: pulled price from sushi lps.

### LINKED ISSUE #684: Pull Prices from Sushi LPs rather than Coingecko AND show them in the frontend
The coingecko price for Ricochet is wrong. This issue is about switching where the prices for tokens are pulled from. Rather than pulling the price from Coingecko, fetch the price from the Sushi liquidity pool. I believe you can get this from Sushiswap's subgraph: https://thegraph.com/hosted-service/subgraph/sushiswap/sushiswap

Additionally, expose this price in the frontend for each market, like this:

Screen Shot 2022-05-26 at 8 29 47 AM

### PR DESCRIPTION
**Pull request by zhyd1997**
Fixes #684

### CODE REVIEW

**Review by mikeghen — CHANGES_REQUESTED** [review]
The price shouldnt show up on the launchpad

Screen Shot 2022-05-27 at 9 44 29 AM

Could you match the style from the launchpad? Which puts the price above the market using a smaller font. Also use `@ 0.03 USDC/RIC` like it is on the launchpad, use @ symbol instead of 🚀 

Include prices for all the markets

Screen Shot 2022-05-27 at 9 44 43 AM

**Review by mikeghen — CHANGES_REQUESTED** [review]
1. Can you make the font a little smaller?
2. It looks like every price is the same
Screen Shot 2022-05-27 at 10 49 29 AM

**Comment by zhyd1997** [comment]
@mikeghen do we need other pair price 👀

**Comment by mikeghen** [comment]
> @mikeghen do we need other pair price 👀

Yes

File: src/components/common/Price/index.tsx [hunk 1]
```diff
@@ -7,46 +7,99 @@ import Web3 from 'web3';
...
 import { selectMain } from '../../../store/main/selectors';
+import { FlowTypes } from 'constants/flowConfig';
+import { Coin } from 'constants/coins';
+import { querySushiPoolPirces } from 'api';
+
+// Sushi Pools
+const poolAddresses = {
+	[`${Coin.RIC}-${Coin.USDC}`]: '0xdbf5d66d77a83b96763c965d193d0fdd1f8a184b', // need {token1Price}
+	[`${Coin.USDC}-${Coin.RIC}`]: '0xdbf5d66d77a83b96763c965d193d0fdd1f8a184b', // need {token0Price}
+	// FIXME: add other pools
```
Comment (zhyd1997) [inline src/components/common/Price/index.tsx]: @mikeghen 

need other liquidity pool contract addresses here.

**Comment by 0xSero** [comment]
Any progress on this @zhyd1997

**Comment by zhyd1997** [comment]
Need @mikeghen response.

**Comment by mikeghen** [comment]
Liquidity pools, some are quickswap:

RIC<>USDC, Sushi: 0xDBF5d66d77a83B96763c965D193D0fdD1f8A184B
ETH<>USDC, Sushi: 0x34965ba0ac2451A34a0471F04CCa3F990b8dea27
WBTC<>USDC, Sushi: 0xD02b870c556480491c70AaF98C297fddd93F6f5C 
MATIC<>USDC, Quickswap: 0x6e7a5FAFcec6BB1e78bAE2A1F0B612012BF14827
ETH<>DAI, Sushi: 0x6FF62bfb8c12109E8000935A6De54daD83a4f39f
WBTC<>DAI, Sushi: 0x7a1d5E67c3a273274766E241363E3E98e721E456
MATIC<>DAI, Quickswap: 0xEEf611894CeaE652979C9D0DaE1dEb597790C6eE

**Comment by mikeghen** [comment]
Looks good but if its 0 like this could we just show `< 0.001 ETH/USDC`?
Screen Shot 2022-06-08 at 2 50 16 PM

**Review by mikeghen — APPROVED** [review]

**Comment by mikeghen** [comment]
good work @zhyd1997 👏

**Pull request merged**